error/ApiError.js

```diff
@@ -0,0 +1,17 @@
+class ApiError {
```

Comment:
The good practice is to have custom error inherited from Error object. Can you please add this?